pkg/virt-launcher/virtwrap/converter/numa_placement.go

```diff
@@ -124,6 +124,11 @@ func numaMapping(vmi *v1.VirtualMachineInstance, domain *api.DomainSpec, topolog
 			domain.CPU.NUMA.Cells[i].Memory += hugepagesSize
 		}
 	}
+	if vmi.IsRealtimeEnabled() {
+		// RT settings when hugepages are enabled
+		domain.MemoryBacking.NoSharePages = &api.NoSharePages{}
+		domain.MemoryBacking.Locked = &api.MemoryLock{}
+	}
```

Comment:
in `staging/src/kubevirt.io/client-go/api/v1/schema.go` there are values defined on the VMI spec related to NoSharePages and Locked, was unintentional?

because here we're not looking at the values on the VMI, but setting these domain values based on the realtimeEnabled setting.

Reply:
It was a leftover from the initial design. I will remove them.